tests/autoyast/installation.pm

```diff
@@ -157,6 +157,10 @@ sub run {
         push(@needles, 'autoyast-license');
     }
 
+    # repo key expired bsc#1179654
+    record_soft_failure 'bsc#1179654';
```

Comment:
As this bug is only for SLE15-GA OS (as there is a `if is_sle('=15')`) it could be better add the `record_soft_failure` in the `if`, no? To avoid lot of false softfailure in 15-SP3 for example.

Reply:
True, fixing.